content/technotes/2017-11-07-magick-knitr.md

```diff
@@ -0,0 +1,111 @@
+---
+slug: "magick-knitr"
+title: "Using Magick with RMarkdown and Shiny"
+date: 2017-11-07
+authors:
+  - name: Jeroen Ooms
+categories: technotes
+cover: "https://i.imgur.com/tTFk7ig.jpg"
+tags:
+- R
+- packages
+- magick
+- images
+---
+
+This week [magick](https://cran.r-project.org/web/packages/magick/vignettes/intro.html) 1.5 appeared on CRAN. The latest update adds support for using images in knitr documents and shiny apps. In this post we show how this nicely ties together a reproducible image workflow in R, from source image or plot directly into your report or application.
```

Comment:
Did you want to link to vignette here? or did you mean to go to cran pkg landing page?

Reply:
Yes the vignette.